Formula/f/freebayes.rb

```diff
@@ -28,73 +28,34 @@ class Freebayes < Formula
-
-  resource "vcflib" do
-    url "https://github.com/vcflib/vcflib/releases/download/v1.0.12/vcflib-1.0.12-src.tar.gz"
-    sha256 "cb1e78d675f06ae3b22ffed7e1dea317e4c7f826e51e5b5dabd80240efbe1019"
-
-    # Backport fix for using external `wfa2-lib`
-    patch do
-      url "https://github.com/vcflib/vcflib/commit/5e4edec2fba5d5a51dae7a9fe48d0252ade53857.patch?full_index=1"
-      sha256 "e7d6d433d837dd823916ef91fe0165bf4ba4f05c52fd4c9517aef7f80653a2a8"
-    end
-
-    # Apply open PR to help find `wfa2-lib` include directory
-    # PR ref: https://github.com/vcflib/vcflib/pull/413
+    # Fix to error: ‘numeric_limits’ is not a member of ‘std’
     patch do
-      url "https://github.com/vcflib/vcflib/commit/9f9237ff0e6b4887f0edfc88587957aa736ced7b.patch?full_index=1"
-      sha256 "bede43d22b4b47141cd90edc4df90f65b9ac41e9598c2b05b2fe7fa84ea51aa8"
+      url "https://github.com/ekg/intervaltree/commit/aa5937755000f1cd007402d03b6f7ce4427c5d21.patch?full_index=1"
+      sha256 "7ae1070e3f776f10ed0b2ea1fdfada662fcba313bfc5649d7eb27e51bd2de07b"
     end
   end
 
-  # Apply open PR to help Meson locate vcflib and wfa2 libraries
-  # PR ref: https://github.com/freebayes/freebayes/pull/822
-  patch do
-    url "https://github.com/freebayes/freebayes/commit/b458396e1acbad3983c70c202a6db2b3711a8eac.patch?full_index=1"
-    sha256 "b5c7d855d4d66c6c96dada307e82ccbf0b6904a25928c4f3e163f52e178b7907"
-  end
-
-  # Apply open PR to fix build when using git submodules
-  # PR ref: https://github.com/freebayes/freebayes/pull/823
-  patch do
-    url "https://github.com/freebayes/freebayes/commit/35eeacb6468fdce25233a33f7216f4e776d381f9.patch?full_index=1"
-    sha256 "1b6f0bb1e369a4b11e9a7754f3b789035b39cc5d9e3dbbac84fde21893f0d9be"
-  end
-
   def install
-    resource("vcflib").stage do
-      rm_r(["contrib/WFA2-lib", "contrib/tabixpp/htslib"]) # avoid bundled libraries
-
-      system "cmake", "-S", ".", "-B", "build",
-                      "-DBUILD_DOC=OFF",
-                      "-DBUILD_ONLY_LIB=ON",
-                      "-DZIG=OFF",
-                      *std_cmake_args(install_prefix: buildpath/"vendor")
-      system "cmake", "--build", "build"
-      system "cmake", "--install", "build"
-      system "make", "-C", "contrib/intervaltree", "install", "DESTDIR=", "PREFIX=#{buildpath}/vendor"
-      (buildpath/"vendor/include").install "contrib/tabixpp/tabix.hpp"
+    # add contrib to include directories
+    inreplace "meson.build", "incdir = include_directories(", "incdir = include_directories('contrib',"
 
-      ENV.append_path "LIBRARY_PATH", buildpath/"vendor/bin"
-      ENV.append_to_cflags "-I#{buildpath}/vendor/include"
-      ENV.append_to_cflags "-I#{buildpath}/vendor/include/vcflib"
-    end
+    # install intervaltree
+    (buildpath/"contrib/intervaltree").install resource("intervaltree")
+    # add tabixpp to include directories
+    ENV.append_to_cflags "-I#{Formula["tabixpp"].opt_include} -L#{Formula["tabixpp"].opt_lib} -ltabix"
```

Comment:
smells like a meson config issue (it cannot find system taxipp)

Reply:
That's right. Upstream uses `find_library("tabixpp")` and so I added it same as `vcflib` before.
I am not familiar with `meson` build.. If there are any workaround, let me know.